sound/soc/sof/intel/hda-dsp.c

```diff
@@ -336,6 +337,11 @@ int hda_dsp_set_power_state(struct snd_sof_dev *sdev,
 	dev_vdbg(bus->dev, "D0I3C updated, register = 0x%x\n",
 		 snd_hdac_chip_readb(bus, VS_D0I3C));
 
+	if (d0_substate == SOF_DSP_D0I3)
+		*flags = SOF_PM_PG_STREAMING;
```

Comment:
This is logically incorrect.

The device can be in D0i3 and e.g. do event detection without streaming PCM, ever.

Reply:
the bit SOF_PM_PG_STREAMING is not used in SOF FW yet, so let me set the whole flag to be 0 for D0i3, and 'SOF_PM_PPG | SOF_PM_PCG' for D0.